corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -275,11 +275,11 @@ def _passes_user_data_filter(self, contact):
             actual_values_set = self.convert_to_set(user_data.get(key, ""))
 
             if actual_values_set.isdisjoint(allowed_values_set):
-                return False
+                return False, f"Filtered out on property {key}: exp: ({','.join(allowed_values_set)}), act: ({','.join(actual_values_set)})"
```

Comment:
nit: I'm assuming `exp` stands for expected? Should we relabel this as "allowed"

Reply:
renamed to "allowed" and "found"